packages/payload/src/fields/validations.ts

```diff
@@ -170,8 +170,8 @@ export const code: Validate<unknown, unknown, CodeField> = (value: string, { req
 }
 
 export const json: Validate<unknown, unknown, JSONField & { jsonError?: string }> = (
-  value: string,
-  { jsonError, required, t },
+  value: string, // is this typing correct?
```

Comment:
Is value really a string (Stringified JSON) or is this a type mistake? Shouldn't it be `any` or `unknown`?

Reply:
I agree, string is wrong. We will probably have to use `unknown` instead.